main.c

```diff
@@ -1,16 +1,17 @@
-
+#include <hal/debug.c>
+#include <hal/video.c>
 #include <stdio.h>
 #include <time.h>
-#include "SDL2/SDL.h"
-#include "SDL2/SDL_image.h"
+#include <SDL.h>
+#include <SDL_image.h>
 
 #if __EMSCRIPTEN__
   #include <emscripten.h>
   #include <SDL/SDL_ttf.h>
   #include <SDL/SDL_mixer.h>
 #else
-  #include "SDL2/SDL_ttf.h"
-  #include <SDL2/SDL_mixer.h>
+  #include <SDL_ttf.h>
+  //#include <SDL2/SDL_mixer.h>
```

Comment:
All this mixer disabling should have been a separate commit or at least a define like: `#define DISABLE_AUDIO` - that way you can quickly check / renable it if it works later.